api/openapi/specs/core_service.yaml

```diff
@@ -0,0 +1,331 @@
+openapi: 3.0.0
+
+info:
+  title: Enclave API
+  description: API to manipulate the enclaves
+  version: 0.1.0
+
+paths:
+
+  /enclaves/{enclave_identifier}/starlark:
+    get:
+      description: Get last Starlark run
+      parameters:
+        - $ref: './api_types.yaml#/components/parameters/enclave_identifier'
+      responses:
+        default:
+          $ref: '#/components/responses/NotOk'
+        "200":
+          description: Successful request
+          content:
+            application/json:
+              schema:
+                $ref: "./api_types.yaml#/components/schemas/StarlarkDescription"
+
+  /enclaves/{enclave_identifier}/starlark/packages:
+    post:
+      description: Uploads a Starlark package. This step is required before the package can be executed with RunStarlarkPackage
+      parameters:
+        - $ref: './api_types.yaml#/components/parameters/enclave_identifier'
+      requestBody:
+        $ref: "./api_types.yaml#/components/requestBodies/fileUploadBody"
```

Comment:
I don't see the fileUploadBody inside the "api_types.yaml" Do I missing something?

Reply:
is this a stream right? so no type definition I guess

Reply:
Ended up merge all specs into a single file. But the definition is now [here](https://github.com/kurtosis-tech/kurtosis/blob/f3cb0c0d363bed2402603420b7e120d146980482/api/openapi/specs/kurtosis_api.yaml#L597)